app/test_split_integer.py

```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_or_the_parts_should_be_equal_to_value():
-    pass
+    actual = split_integer(17, 4)
+    assert sum(actual) == 17
 
 
 def test_should_split_into_equal_parts_when_value_is_divisible_by_parts():
-    pass
+    actual = split_integer(16, 4)
+    assert actual == [4, 4, 4, 4]
```

Comment:
```suggestion
    assert split_integer(16, 4) == [4, 4, 4, 4]
```

Reply:
It's better to use an explicit comparison

Reply:
Or should I change these conditions in all tests? In the training videos, Danya says that this is the best practice.